tasks/manpages.rake

```diff
@@ -1,21 +1,21 @@
-# require 'fileutils'
+# This is borrowed from `lib/puppet/face/man`, which uses the ERB
+# template in the help application to render the man page
+def generate_ronn_from_face(face)
+  # the erb template references the `face` application
+  file = File.join(RAKE_ROOT, 'lib/puppet/face/help/man.erb')
+  erb = ERB.new(File.read(file), nil, '-')
+  erb.filename = file
+  return erb.result(binding)
+end
```

Comment:
might be better to push this into `puppet help --man <app>` so that the rake task and docs can use it, while eliminating the need for `puppet man`

Reply:
Updated